hypha/templates/base.html

```diff
@@ -211,5 +211,25 @@ <h4 class="heading heading--no-margin">@{{ settings.utils.SocialMediaSettings.tw
         <script src="{% static 'js/main.js' %}"></script>
         <script src="{% static 'js/cookieconsent.js' %}"></script>
         {% block extra_js %}{% endblock %}
+
+        {% block sentry_sdk %}
+        {% if SENTRY_PUBLIC_KEY %}
+        <script
+            src="https://js.sentry-cdn.com/{{ SENTRY_PUBLIC_KEY }}.min.js"
+            crossorigin="anonymous"
+        ></script>
+        <script>
+            Sentry.onLoad(function() {
+                Sentry.init({
+                    environment: "{{ SENTRY_ENVIRONMENT }}",
+                    debug: {% if SENTRY_DEBUG %}true{% else %}false{% endif %},
+                    integrations: [new BrowserTracing()],
+                    denyUrls: {{ SENTRY_DENY_URLS|safe }},
+                    tracesSampleRate: {{ SENTRY_TRACES_SAMPLE_RATE }}
+                });
+             });
+        </script>
+        {% endif %}
+        {% endblock sentry_sdk %}
```

Comment:
Is this not also needed in `base-apply.html`?

Should you have the block name in the end tag?

Reply:
1. yes. I always forget to add it to the base-apply.html
2. block name, in the end, is allowed, and make things clear which block is getting terminated.

Reply:
Added.